src/base-styles/reset.scss

```diff
@@ -0,0 +1,192 @@
+/**
+* reset.css
+* Custom normailze/reset
+*
+* - Normalizes styling across browsers
+* - Sets sensible defaults for base elements
+* - Repeats some key rules from `semantic-ui` that allow us to remove it from consumers
+*/
+
+// border-box on all elements
+html {
+  box-sizing: border-box;
```

Comment:
This seemed like the riskiest part of this PR, but looking again this is set in semantic-ui anyway, and our overrides were only to adjust for that in Storybook.